house-snark/src/relations/shielder/types.rs

```diff
@@ -149,6 +148,25 @@ mod casting {
         }
     }
 
+    macro_rules! to_bytes_through_backend {
+        ($frontend_type:ty, $backend_type:ty) => {
```

Comment:
`backend_type:ty` is not even used, right?

Reply:
true, removed